libvirt/tests/src/virtual_network/qemu/packet_stress_test.py

```
@@ -53,93 +53,61 @@ def run(test, params, env):
+        test.log.debug("TEST_STEP: Check guest work normally and no crash occurred.")
+        vm.verify_kernel_crash()
+        vm.verify_dmesg()
 
     iface_type = params.get("iface_type")
-    basic_iface_type = params.get("basic_iface_type", "network")
+    basic_iface_type = params.get("basic_iface_type", "bridge")
+    vm_name = params.get("main_vm")
+    vm = env.get_vm(vm_name)
 
-    iface_list = eval(params.get('iface_list', '[]'))
     iface_list_2 = eval(params.get('iface_list_2', '{}'))
```

Ruff (0.14.8)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)